build-data-capturing-maven-samples/capture-ge-extension-version/maven-ge-extension-version.groovy

```diff
@@ -0,0 +1,9 @@
+import com.gradle.maven.extension.api.scan.BuildScanApi
+
+/**
+ * This Groovy script captures the Gradle Enterprise Maven Extension version as a custom value.
+ */
+
+InputStream stream = BuildScanApi.class.classLoader.getResourceAsStream("com.gradle.scan.plugin.internal.meta.buildAgentVersion.txt")
```

Comment:
We cannot use buildScanApi.class.classLoader.getResource here as we do it for Gradle?

Reply:
What I meant is to use the `getResource` API for in both scripts, instead of suing `getResource` in one and `getResourceAsStream` in the other.

Reply:
Great point. I have no idea why I went down this route thinking they cannot access the same built-in groovy APIs. They are now updated to use the same API.